packages/manager/apps/dedicated/client/app/account/user/dashboard/user-dashboard.html

```diff
@@ -97,6 +97,7 @@
                         <oui-action-menu data-compact data-placement="end">
                             <oui-action-menu-item
                                 data-href="#/useraccount/infos"
+                                top="true"
```

Comment:
Seems to redirect from `dedicated` to `dedicated`. 
Not external and not `buildURL`.